source/includes/_tipser-tools.md

```diff
@@ -1,3 +1,35 @@
 # Tipser Tools
 
-//tools desc
\ No newline at end of file
+Tipser enables commerce on almost any site. An easy way to enable shopping on your site is to install the Tipser script and use the Tipser Tools to insert `Get The Look` widget directly into your content. The installation of the script on your site typically takes less than 5 minutes, and can be done by even non-technical people. 
+
+To get Tipser Tools visit [www.tipser.com/pos/tools](https://www.tipser.com/pos/tools) and follow the instructions there.
```

Comment:
The page that you are linking to is private (requires authentication). Maybe we need to mention that it's needed to log in to your POS account?

Reply:
this is some good idea!